### DISCUSSION TITLE
Is there any possibility to rename View?

### DISCUSSION THREAD

**Discussion by LoveYourMommy** [directvt/vtm#179] [Q&A] [answered]
Is there any possibility to rename Views?

**Comment by o-sdn-o** [comment, marked as answer]
At the moment ~~it is not possible~~ to rename them. I opened a new issue #180 to track this request.

**Comment by o-sdn-o** [comment]
Here is an idea how to change view titles using clipboard and right click

https://user-images.githubusercontent.com/11535558/166443498-9e222f69-2eae-4560-a4fd-068518790bfd.mp4
